core/org.wso2.carbon.ui/src/main/java/org/wso2/carbon/ui/filters/cache/AbstractCachePreventionFilter.java

```diff
@@ -134,12 +134,13 @@ protected void initPatternsAndAction(FilterConfig filterConfig) {
 
         String patternsParam = filterConfig.getInitParameter(PARAM_NAME_PATTERNS);
 
-        //If a filtering pattern is available split the pattern using carriage return and line feed  characters and put into an array.
+        //If a filtering pattern is available split the pattern using carriage return and line feed characters and put into an array.
```

Comment:
Keep a space after the comment ` If a filtering`

Reply:
Fixed with https://github.com/wso2/carbon-kernel/pull/3322